docs/source/Diff.rst

```diff
@@ -5,7 +5,9 @@ Job Diff
 ========
 
 Avocado Diff plugin allows users to easily compare several aspects of
-two given jobs. The basic usage is::
+two given jobs. The basic usage is:
+
+.. code-block:: diff
```

Comment:
I'll check how this renders, because there's both the command and the "diff like" output. Supposing you've tried it, it's a nice idea.